google-cloud-spanner/src/test/java/com/google/cloud/spanner/IntegrationTestEnv.java

```diff
@@ -58,8 +58,15 @@ public class IntegrationTestEnv extends ExternalResource {
   private InstanceAdminClient instanceAdminClient;
   private DatabaseAdminClient databaseAdminClient;
   private boolean isOwnedInstance;
+  private boolean useNewInstance;
```

Comment:
nit: Maybe rename to `alwaysUseOwnedInstance`?

Reply:
Thanks Knut. It seems like a bit confusing with `isOwnedInstance`, can we use `createNewInstance`?

Reply:
Could we in that case use `alwaysCreateNewInstance`. My point is that any of these tests __could__ create a new instance. I think it will be confusing if the test creates a new instance while the property `createNewInstance` is `false`. The `always` prefix indicates that it forces the specific test to always create a new instance, regardless whether the test environment has defined an existing instance name that should be used.